packages/astro/src/build/index.ts

```diff
@@ -131,7 +131,7 @@ class AstroBuilder {
         rollupPluginHTML({
           input,
           extractAssets: false,
-        }),
+        }) as any, // CI fix: ignore typing of this plugin
```

Comment:
Fix: TypeScript doesn’t need to worry about typings for an actively-maintained plugin. Also this is just the initializer; the settings within are still typechecked and have autocomplete!